standard/lexical-structure.md

```diff
@@ -262,19 +274,19 @@ fragment Input_Character
     // anything but New_Line_Character
     : ~('\u000D' | '\u000A'   | '\u0085' | '\u2028' | '\u2029')
     ;
-    
+
```

Comment:
Were these space removals deliberate, or just something your editor did? I wonder whether it might be worth removing them from this PR, and then (if we want to) running something to remove them from the whole spec. (My guess is there will be other occurrences.)

Reply:
(not related to this PR, but this comment)
I'm fine removing them as we find them. There are many places in code and grammar where multiple spaces are intended. I think the sweep will be painful to get right.